### PR TITLE
post: A journey through your code

### PR DESCRIPTION
**Pull request by tobiastimm**
This is a repost, of my own original post.
https://dev.to/tobiastimm/onboard-your-codebase-with-codetour-2jc8

The introduction is rewritten

I can take my own post down if this gets published :)

### CODE REVIEW

File: posts/tobi/a-journey-through-your-code.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: A Journey through your Code
+published: false
+description: Create exciting onboardings with CodeTour
+tags: 'onboarding, vscode, resource, codenewbie'
```
Comment (kotzendekrabbe) [inline posts/tobi/a-journey-through-your-code.md:5]: recommendation for hashtags on the existing and moderate hashtags of dev.to

resource -> productivity

File: posts/tobi/a-journey-through-your-code.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+cover_image: posts/assets/000_cover.jpg
```
Comment (kotzendekrabbe) [inline posts/tobi/a-journey-through-your-code.md:6]: delete the cover_image if you don't need it

File: posts/tobi/a-journey-through-your-code.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+---
```
Comment (kotzendekrabbe) [inline posts/tobi/a-journey-through-your-code.md:1]: great one - and something new I learned :) thank you

**Review by kotzendekrabbe — CHANGES_REQUESTED** [review]

File: posts/tobi/a-journey-through-your-code.md (hunk 2 above)
Comment (tobiastimm) [reply posts/tobi/a-journey-through-your-code.md]: fixed

File: posts/tobi/a-journey-through-your-code.md (hunk 1 above)
Comment (tobiastimm) [reply posts/tobi/a-journey-through-your-code.md]: changed! ty

File: posts/tobi/a-journey-through-your-code.md (hunk 3 above)
Comment (tobiastimm) [reply posts/tobi/a-journey-through-your-code.md:1]: Fixed your comments 👍

**Review by kotzendekrabbe — APPROVED** [review]

**Pull request merged**